dev/com.ibm.ws.microprofile.config_fat/test-applications/simultaneousRequests.war/resources/META-INF/permissions.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<permissions xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/permissions_7.xsd"
+	version="7">
+	<permission>
+		<class-name>java.security.AllPermission</class-name>
+		<name>*</name>
+		<actions>*</actions>
+	</permission>
```

Comment:
We shouldn't need to grant all permissions in _any_ test app...

Reply:
Though it may be fair for now if we don't have it documented which permissions are needed to run with Config etc.